README.md

```diff
@@ -108,6 +108,7 @@ Chemfiles can read and write files in the following formats:
 - TRIPOS mol2 files;
 - MacroMolecular Crystallographic Information Framework (CIF) files;
 - MacroMolecular Transmission Format (MMTF) files;
+- Daylight SMILES: .smi;
```

Comment:
Since we are implementing OpenSMILES, should this be the name we use?

Reply:
`Daylight SMILES` is what most people who use SMILES refer to it as. I used OpenSMILES as a link because their description of the format is more cohesive and complete. We technically can't call our parser `OpenSMILES` because we don't check to see if the aromaticity is correct and we accept whatever the user gives.